packages/backend/src/api/endpoints/IdPOAuthAccessTokenApi.ts

```diff
@@ -5,11 +5,11 @@ import { AbstractAPI } from './AbstractApi';
 const basePath = '/oauth_applications/access_tokens';
 
 export class IdPOAuthAccessTokenApi extends AbstractAPI {
-  async verifySecret(secret: string) {
+  async verifyAccessToken(accessToken: string) {
```

Comment:
The machine token BAPI are not exposed in the SDK yet so it's safe to update this